pkg/eden/eden.go

```diff
@@ -341,7 +341,7 @@ func StatusEServer() (status string, err error) {
 }
```

Comment:
Same large comment - let's separate the functions and implementation-specific things. 
I'd like to see 
  Vbox.go
  Parallels.go
  Qemu.go

Reply:
will do that

Reply:
made separate files